packages/eslint-plugin-pf-codemods/src/rules/v6/tokensWarn/tokens-warn.ts

```diff
@@ -57,3650 +59,24 @@ module.exports = {
-  "global_palette_black_200",
-  "global_palette_black_300",
-  "global_palette_black_400",
-  "global_palette_black_500",
-  "global_palette_black_600",
-  "global_palette_black_700",
-  "global_palette_black_800",
-  "global_palette_black_850",
-  "global_palette_black_900",
-  "global_palette_blue_100",
-  "global_palette_blue_200",
-  "global_palette_blue_300",
-  "global_palette_blue_400",
-  "global_palette_blue_50",
-  "global_palette_blue_500",
-  "global_palette_blue_600",
-  "global_palette_blue_700",
-  "global_palette_cyan_100",
-  "global_palette_cyan_200",
-  "global_palette_cyan_300",
-  "global_palette_cyan_400",
-  "global_palette_cyan_50",
-  "global_palette_cyan_500",
-  "global_palette_cyan_600",
-  "global_palette_cyan_700",
-  "global_palette_gold_100",
-  "global_palette_gold_200",
-  "global_palette_gold_300",
-  "global_palette_gold_400",
-  "global_palette_gold_50",
-  "global_palette_gold_500",
-  "global_palette_gold_600",
-  "global_palette_gold_700",
-  "global_palette_green_100",
-  "global_palette_green_200",
-  "global_palette_green_300",
-  "global_palette_green_400",
-  "global_palette_green_50",
-  "global_palette_green_500",
-  "global_palette_green_600",
-  "global_palette_green_700",
-  "global_palette_light_blue_100",
-  "global_palette_light_blue_200",
-  "global_palette_light_blue_300",
-  "global_palette_light_blue_400",
-  "global_palette_light_blue_500",
-  "global_palette_light_blue_600",
-  "global_palette_light_blue_700",
-  "global_palette_light_green_100",
-  "global_palette_light_green_200",
-  "global_palette_light_green_300",
-  "global_palette_light_green_400",
-  "global_palette_light_green_500",
-  "global_palette_light_green_600",
-  "global_palette_light_green_700",
-  "global_palette_orange_100",
-  "global_palette_orange_200",
-  "global_palette_orange_300",
-  "global_palette_orange_400",
-  "global_palette_orange_50",
-  "global_palette_orange_500",
-  "global_palette_orange_600",
-  "global_palette_orange_700",
-  "global_palette_purple_100",
-  "global_palette_purple_200",
-  "global_palette_purple_300",
-  "global_palette_purple_400",
-  "global_palette_purple_50",
-  "global_palette_purple_500",
-  "global_palette_purple_600",
-  "global_palette_purple_700",
-  "global_palette_red_100",
-  "global_palette_red_200",
-  "global_palette_red_300",
-  "global_palette_red_400",
-  "global_palette_red_50",
-  "global_palette_red_500",
-  "global_palette_white",
-  "global_primary_color_100",
-  "global_primary_color_200",
-  "global_primary_color_dark_100",
-  "global_primary_color_light_100",
-  "global_secondary_color_100",
-  "global_spacer_form_element",
-  "global_success_color_100",
-  "global_success_color_200",
-  "global_target_size_MinHeight",
-  "global_target_size_MinWidth",
-  "global_warning_color_100",
-  "global_warning_color_200",
-];
+// consumers may run class-name-updater before codemods, so we have to check also old tokens with v6 prefix
```

Comment:
Is this still applicable after changes made in https://github.com/patternfly/pf-codemods/pull/754 ?

Reply:
> I would rather keep it there if someone has already run the old version of class-name-updater

I'm good with keeping it for this